website/docs/components/form/primitives/partials/guidelines/guidelines.md

```diff
@@ -1,47 +1,43 @@
+We recommend using a combination of client-side and server-side validation for the best user experience.
 
-##### Client side validation
+### Client-side validation
 
-Client side validation, sometimes also referred to as inline validation, is an initial check that happens in the browser to ensure required fields are filled out and that the value is in the correct format. Learn more about [client side validation](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation).
+[Client-side validation](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation), sometimes called inline validation, is an initial check in the browser to ensure that required fields aren’t empty and the value is in the correct format. Client-side validation allows the user to quickly resolve errors before submitting the form.
 
-![Example of client side form validation](/assets/components/form/primitives/form-validation-client.png)
+![Example of client side form validation](/assets/components/form/primitives/form-primitives-validation-client.png =600x*)
 
-##### Server side validation
+### Server-side validation
 
-Server side validation provides a more thorough check on the server once the data has been submitted and helps keep our applications safe.
+Server-side validation provides a more thorough check on the server once the data has been submitted and helps keep our applications safe.
 
-When using server side validation, display an `AlertInline` in the `Critical` variant above the form listing **all** errors with links to each invalid field.
+When using server-side validation, display a Critical Inline [Alert](/components/alert) above the form listing **all** errors with links to each invalid field.
 
-![Example of server side form validation](/assets/components/form/primitives/form-validation-server.png)
+![Example of server side form validation](/assets/components/form/primitives/form-primitives-validation-server.png =600x*)
 
 ## Content
 
 ### Form::Label
 
 - We recommend keeping labels clear and concise, about 1-3 words. They should not consist of full sentences.
-- [3.3.2 Labels or Instructions (A)](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html): Labels or instructions are provided when content requires user input.
+- [3.2.2 Labels or Instructions (A)](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html): Labels or instructions are provided when content requires user input.
 
 ### Form::HelperText
 
-- Use `Form::HelperText` when needing to provide the user with extra details about the data you’re asking them to input, eg. formatting requirements such as MM-DD-YYYY.
+- Use helper text to give the user extra details about the data you’re asking them to input, e.g., formatting requirements such as MM-DD-YYYY.
```

Comment:
Another broader writing guideline we should define, when talking about examples what abbreviation do we use? I've seen  e.g., i.e., ex. there are probably a couple of others too

Reply:
I looked into this a little bit and determined "e.g.," usually covers most of our use cases. It's important to note that they are not the same though so we should strive to use them correctly, not interchangeably. 

I read through Grammarly's article on [the difference between ie and eg](https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/?gclid=CjwKCAiA2rOeBhAsEiwA2Pl7Q4Y46cv5f9TVNManxUhoRKjTZ0OkdYxv4PwRc-SzUP1zTFfWecYs-RoCymIQAvD_BwE&gclsrc=aw.ds), in case you're curious.